megalinter/reporters/AzureCommentReporter.py

```diff
@@ -2,15 +2,6 @@
 """
 Azure Comment reporter
 Post a comment on Azure Pipelines Pull Requests
-
-Requires the following vars sent to docker run:
-- SYSTEM_ACCESSTOKEN
-- SYSTEM_COLLECTIONURI
-- SYSTEM_PULLREQUEST_PULLREQUESTID
-- SYSTEM_PULLREQUEST_SOURCEREPOSITORYURI
-- SYSTEM_TEAMPROJECT
-- BUILD_BUILDID
-- BUILD_REPOSITORY_ID
```

Comment:
Did it degrade the published docs? Otherwise, that info is kinda relevant, or maybe a note to refer at the docs for that info (that is converted to a docker run example, not as a requirement like it is now)

Reply:
@echoix I removed it from there and moved it to markdown because the way it was, it was hidden since you had to go to the repository and that particular class to see those instructions, which didn't make sense.

The rest of the similar reporters (bitbucket, gitlab, github) don't have any notes in the code and they do have markdown instructions. What I have done is to leave it homogeneous with the rest.

Reply:
Ok, perfect then!